installer.sh

```diff
@@ -409,7 +409,7 @@ echo "==========================================================================
 mkdir -p /etc/keylime
 mkdir -p config
 pushd scripts
-python3 convert_config.py --input ../keylime.conf --out ../config
+python3 convert_config.py --input /dev/null --out ../config
```

Comment:
Use `--input` as optional, and it will default to `None`.  This can signal what you want, and the cli will be clear: `python3 convert_config --out ../config`

Reply:
The default behavior when no input is provided is to use the system installed configuration files as input to generate upgraded config without changing the user set options. This is already implemented.

What I want is to use the default values for the options ignoring any eventual system installed files. The workaround was to provide an empty file as input (so that it wouldn't search for the system installed files). But it is kind of ugly.

I decided to add a separate option `--defaults` to make the script to ignore any input.